lib/Bio/EnsEMBL/DataCheck/Checks/CheckConservationScore.pm

```diff
@@ -0,0 +1,73 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckConservationScore;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckConservationScore',
+  DESCRIPTION    => 'The MLSS for GERP_CONSERVATION_SCORE should have conservation score entries',
+  GROUPS         => ['compara', 'compara_pairwise_alignments'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['conservation_score', 'genomic_align_block', 'method_link', 'method_link_species_set', 'method_link_species_set_tag']
```

Comment:
If it is used by the adaptors it does not **need** to be declared, there will be no complaints. DCs will complain if a table is listed and not present in a DB though. I have stuck to it because I suppose it is good practice, will make altering affected DCs based on schema changes/API changes (although likely rare) easier, and if more special things are included in the DC framework around the tables, it's already here

Reply:
There is a pipeline that can detect changes in the tables and only rerun the relevant DCs. It relies on this list of tables

Reply:
The `tables` are part of the mechanism for checking whether a datacheck needs to be re-run - the table's last-updated timestamp is compared to the last successful run of the datacheck. If nothing has changed, then there's no point running the datacheck.
So, if you don't include table that are implicit in the adaptors, you run the risk of not running the datacheck if changes to those tables introduce bugs - because the datacheck won't think it needs to run.